server/eventlog/eventlog.go

```diff
@@ -32,8 +33,16 @@ const (
 	numReadWorkers = 16
 )
 
-func GetEventLogPathFromInvocationId(invocationId string) string {
-	return invocationId + "/chunks/log/eventlog"
+var (
+	//Id of an empty log
+	EmptyId = chunkstore.ChunkIndexAsStringId(chunkstore.EmptyIndex)
+)
+
+func GetEventLogPathFromInvocationIdAndAttempt(invocationId string, attempt uint64) string {
+	if attempt == 0 {
+		return invocationId + "/chunks/log/eventlog"
+	}
```

Comment:
The special-casing of the 0 functionality is to allow for this function to work with old invocations as well?

(if so, can you add a comment so we don't accidentally mess with this?)

Reply:
Yep, and will do.